output/tables/demographics.csv

```diff
@@ -1,5362 +1,5413 @@
+MX_CHH,3556574,1867866,1934667,,,,,,,663048,654615,634027,555333,541787,354274,228819,120297,40266,10067
+MX_CHP,5217908,2927600,3106127,,,,,,,1293325,1186067,1043126,854095,684858,443068,293004,157529,62924,15731
+MX_CMX,8918653,4309255,4801923,,,,,,,1096109,1214964,1504925,1446574,1394233,1073982,773347,389703,173873,43468
+MX_COA,2954915,1652187,1692130,,,,,,,582058,579526,541565,505487,470408,317062,204360,96381,37976,9494
+MX_COL,650555,419037,429715,,,,,,,138716,137046,149755,132460,115798,81769,53635,26080,10794,2699
+MX_DUR,1754754,940248,978267,,,,,,,352781,342369,327232,265337,251297,171212,113976,61636,26140,6535
+MX_GRO,3533251,1920001,2083578,,,,,,,773470,774394,653648,534345,457634,344214,241604,143130,64912,16228
+MX_GUA,5853677,3326463,3587127,,,,,,,1243257,1260405,1217533,1026748,884078,578971,379627,202670,96241,24060
+MX_HID,2858359,1780600,1935066,,,,,,0.723,643452,645279,607481,559331,501853,342854,229746,120819,51881,12970
+MX_JAL,7844830,4639924,4845184,,,,,,,1693898,1660080,1639307,1445967,1252858,811988,554322,280667,116817,29204
+MX_MEX,16187608,8965967,9594354,,,,,,,3038663,3114777,3201490,2928386,2667351,1766153,1101065,502937,191599,47900
+MX_MIC,4584471,2547394,2747028,,,,,,,967129,935148,899706,745111,651485,469928,328354,182987,91659,22915
+MX_MOR,1777227,1044254,1128902,,,,,,,345961,362189,374217,312920,292763,213811,144721,79922,37322,9330
+MX_NAY,1223797,716191,731777,,,,,,,262212,243977,244323,214100,187490,128400,88892,50375,22559,5640
+MX_NLE,5189970,5067660,5157168,,,,,,,2008378,1759562,1732131,1971966,1343652,714876,412564,193263,70749,17687
+MX_OAX,3967889,2158685,2395130,,,,,,,832253,824846,727517,630434,552495,408604,290142,180019,86004,21501
+MX_PUE,6168883,3208410,3532121,,,,,,,1222206,1243418,1154368,978694,838775,573655,387293,216348,100619,25155
+MX_QUE,2038372,1190068,1261563,,,,,,,425965,425030,450164,395176,325870,208617,126949,61275,26068,6517
+MX_ROO,1325578,908638,908956,,,,,,,316539,299038,367315,317907,253258,140378,76709,31881,11655,2914
+MX_SIN,2966700,1533271,1581083,,,,,,,509963,544641,506967,446965,428577,302544,208451,113508,42190,10548
+MX_SLP,2717820,1446750,1544147,,,,,,,514982,546517,487954,421164,385650,271242,188899,110280,51367,12842
+MX_SON,2850330,1629672,1663482,,,,,,,558627,568291,542155,498518,457772,321573,199807,101096,36252,9063
+MX_TAB,2395272,1382356,1447007,,,,,,,508509,474986,496375,448197,377096,251774,155433,78164,31063,7766
+MX_TAM,3441698,1898159,1966886,,,,,,,666599,644963,622259,589656,558559,366191,232587,121280,50361,12590
+MX_TLA,1272847,679525,729504,,,,,,,248193,249279,245332,214617,183554,124746,75586,41797,20740,5185
+MX_VER,8112505,4264507,4600397,,,,,,,1444223,1492677,1446997,1257313,1215672,880527,615945,336099,140361,35090
+MX_YUC,2097175,1120650,1169014,,,,,,,369685,379581,412907,352100,301584,207009,147980,75264,34843,8711
+MX_ZAC,1690868,871283,917456,,,,,,,332445,317436,296133,249683,229689,155406,105720,64103,3049
```

Comment:
The biggest discrepancy I've noticed between total and male/female population...12x for Vatican City.

Reply:
It's a very small country :-) We do what we can with this methodology, what do you propose?

Reply:
Let's handle it consistently with other countries -- I'll comment in another thread on this.